gradle/libs.versions.toml

```diff
@@ -20,7 +20,8 @@ sonar-plugin-api = { group = "org.sonarsource.api.plugin", name = "sonar-plugin-
 sonar-analyzer-commons = { group = "org.sonarsource.analyzer-commons", name = "sonar-analyzer-commons", version.ref = "analyzer-commons" }
 sonar-analyzer-test-commons = { group = "org.sonarsource.analyzer-commons", name = "sonar-analyzer-test-commons", version.ref = "analyzer-commons" }
 slang-api = { group = "org.sonarsource.slang", name = "slang-api", version.ref = "slang-dependencies" }
-slang-checks = { group = "org.sonarsource.slang", name = "slang-checks", version.ref = "slang-dependencies" }
+slang-antlr = { group = "org.sonarsource.slang", name = "slang-antlr", version.ref = "slang-dependencies" }
+slang-testing = { group = "org.sonarsource.slang", name = "slang-testing", version.ref = "slang-dependencies" }
```

Comment:
It's temporary, after implementing https://sonarsource.atlassian.net/browse/SONARGO-106 those dependencies will be not needed anymore.